lib/elements/basic-element.html

```diff
@@ -0,0 +1,128 @@
+<!--
+@license
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../utils/boot.html">
+<link rel="import" href="../utils/mixin.html">
+<link rel="import" href="../mixins/properties-changed.html">
+
+<script>
+(function() {
+  'use strict';
+
+  /**
+   * Custom element base class that provides minimal starting point
+   * using PropertyAccessors to create properties.
+   *
+   * @customElement
+   * @polymer
+   * @memberof Polymer
+   * @constructor
```

Comment:
These couple bits (`@constructor`, `@implements`, `@extends`) have to be moved down to `basicElementBase` as I described below.